CODEOWNERS

```diff
@@ -2,13 +2,13 @@
 * @astarche @mgumble
 
 # Owners for the build files
-/CMakeLists.txt      @SparkingSpork @astarche
-/.github/**/*.yml    @SparkingSpork @astarche
-/scripts/            @SparkingSpork @astarche
+/CMakeLists.txt      @SparkingSpork @astarche @mgumble
```

Comment:
Ah my bad. So these are like overrides?

Reply:
Yeah, the last pattern in CODEOWNERS that a given file matches adds the owners listed for that pattern.